docs/source/user_guide.rst

```diff
@@ -167,6 +167,37 @@ we can now retrieve the position of the ISS after some time::
 For more advanced propagation options, check out the
 :py:mod:`poliastro.twobody.propagation` module.
 
+Accounting non-keplerian orbits: perturbation
+---------------------------------------------
+
+poliastro has recently got the module accounting perturbations
+to the Newton gravitational law. The perturbation acceleration
+should be passed to the ``cowell`` propagation method along
+with the parameters describing the disturbing force. For instance,
+let us examine the effect of J2 perturbation::
+
+    >>> from poliastro.twobody.perturbations import J2_perturbation
+    >>> from poliastro.twobody.propagation import cowell
+    >>> from poliastro.twobody.rv import rv2coe
+    >>> r0 = np.array([-2384.46, 5729.01, 3050.46])  # km
+    >>> v0 = np.array([-7.36138, -2.98997, 1.64354])  # km/s
+    >>> orbit = Orbit.from_vectors(Earth, r0 * u.km, v0 * u.km / u.s)
+    >>> tof = (48.0 * u.h).to(u.s).value
+    >>> r, v = cowell(orbit, tof, ad=J2_perturbation, J2=Earth.J2.value, R=Earth.R.to(u.km).value)
```

Comment:
Instead of using the low level `cowell` function directly, can we `orbit.propagate` with `method=cowell` and the rest of the extra parameters?